mapbox/dr-ui
# The `edit.css` prop is optional when displaying `Edit` in `CodeSnippet`

This PR fixes `CodeSnippet` to allow the `Edit` buttons to appear if the `edit.css` is undefined.


## How to test

1. npm ci
2. npm start
3. See testcase "CodeSnippet with edit buttons without CSS" in /CodeSnippet

## QA checklist
Not applicable.

## Before merge
